test/core/transport/chttp2/flow_control_fuzzer.cc

```diff
@@ -193,9 +194,9 @@ void FlowControlFuzzer::Perform(const flow_control_fuzzer::Action& action) {
         send_from_remote.ack_initial_window_size =
             sent_to_remote.initial_window_size;
         for (const auto& id_stream : streams_) {
-          GPR_ASSERT(id_stream.second.window_delta +
-                         *sent_to_remote.initial_window_size <=
-                     (1u << 31) - 1);
+          CHECK(id_stream.second.window_delta +
+                    *sent_to_remote.initial_window_size <=
+                (1u << 31) - 1);
```

Comment:
Is it then better to use CHECK_LE here (and similar for other places) for the same reason?